docs/troubleshooting/api/client-server_errors_during_the_api_call.md

```diff
@@ -0,0 +1,96 @@
+---
+keywords: ['api', 'error', 'client']
+slug: /client-server-errors-during-the-api-call
+title: Client-Server Errors during the API call
+---
+# Client-Server Errors During the API Call
+
+
+When calling an API in FlutterFlow, you may run into client-server errors. These typically come as status codes that indicate what went wrong—either on your end (the client) or on the server you're requesting data from.
+
+This guide will help you understand the most common API error codes and how to fix them.
+
+## What is an API?
```

Comment:
Remove this sub topic and add a single line about APIs such as To learn more about APIs, check out our API documentation guide and point it to our API docs. You can find it if you search for it. Please reference other guides to learn how we hyperlink internal docs.